### ISSUE TITLE
Bug: Erratic counting on number of messages sent

### ISSUE THREAD

**Issue by mrengy** [ProgressiveCoders/Spoke#1783]
**Describe the bug**
When sending many messages back to back, the counter for number of messages sent will occasionally revert to a lower number. 

**To Reproduce**
1. Get set up as an end user / volunteer who is assigned messages to send 
2. Click the button to send a message repeatedly and watch the counter for messages sent
3. Occasionally the counter will drop to a lower number than what it was before. 

**Expected behavior**
The counter just increments 1 at a time. 

**Screenshots**

**Platform (please complete the following information):**
 - OS: Mac OS 10.15.4
 - Browser : Firefox 
 - Desktop

**Additional context**
I experienced this as a first time volunteer using the platform. The volunteer coordinator also mentioned that this happens sometimes.

**Comment by ibrand** [comment]
Thanks for reporting this @mrengy ! Do you happen to know if this is happening with dynamic or manual assignment? As a way to figure this out: when you're texting, are you able to continually send texts? Or do you need to message your admin and ask for a specific number?

**Comment by mrengy** [comment]
Thanks for looking into it. I’m able to continuously send texts. 

--

**Comment by ibrand** [comment]
Ok to add a little clarity, this is about dynamic assignment. @mrengy do you happen to know what Spoke version you're texting on?

**Comment by mrengy** [comment]
No, I don’t know that. It’s set up through the Rhode Island Working Families Party, in case that helps track down the version. 

--

**Comment by deasterdaywfp** [comment]
Thank you for reporting this @mrengy -- this is behavior common in versions up to 8.0 while using dynamic assignment from our experience. WFP will be moving towards rolling out 9.1 in the next few days/ weeks and may see a change in this behavior on the message counter.

**Comment by Frydafly** [comment]
Checking in whether this was resolved in later versions? @deasterdaywfp

**Comment by arena** [comment]
If this is no longer a bug maybe this issue can be closed. It doesn't seem super easy to reproduce ... has anyone seen it lately?
